packages/babel-preset-es2015/src/index.js

```diff
@@ -1,42 +1,50 @@
 module.exports = function(context, opts) {
-  /* eslint-disable no-var */
-  var loose = false;
-  var modules = true;
-  /* eslint-enable no-var */
+  const moduleTypes = ["commonjs", "amd", "umd", "systemjs"];
+  let loose = false;
+  let modules = "commonjs";
+  let spec = false;
```

Comment:
Sorry I missed this - *i* need this option, as I always want spec mode at all times without question. Correctness > > > speed imo.

My preference would be that spec mode is enabled by default, but lacking that, an easy way to enable it would be really nice :-/